name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SantanderConsumerBankPDFExtractor.java

```diff
@@ -206,56 +207,54 @@ private void addKontoauszugDocument()
         });
         this.addDocumentTyp(type);
 
-        addKontoauszugTaxesTransaction(type);
-        addKontoauszugInterestTransaction(type);
-        addKontoauszugDepositTransaction(type);
-    }
+        // 19.05.2023 19.05.2023 34,00 34,01
+        final String accountStatementBaseRegex = "^(?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) ([\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) (?<amount>[\\.,\\d-]+) ([\\.,\\d-]+) ";
```

Comment:
please adjust this structure to all the other pdf importer.... fintech, dab, comdirec, dkb... ... ...